alternator/server.cc

```diff
@@ -379,7 +380,7 @@ future<executor::request_return_type> server::handle_api_request(std::unique_ptr
     auto units = co_await std::move(units_fut);
     assert(req->content_stream);
     chunked_content content = co_await httpd::read_entire_stream(*req->content_stream);
-    co_await verify_signature(*req, content);
+    auto username = co_await verify_signature(*req, content);
```

Comment:
As far as I can tell, this "username" variable is not used later, so it's weird. I guess you'll use it in the following patch. I hope this won't break bisection by making this commit fail compilation because of an unused variable (I don't remember if we made this a compilation warning or error...).

Reply:
I'm 99% sure I verified that each patch compiled without issues, but I'll double-check

Reply:
Actually, regardless - it doesn't make sense to name the variable here. I'll move the initialization to the patch when it's actually used, and leave this line untouched before that.